doc/adrs/XXX_add_table_description_in_view_dsl.adoc

```diff
@@ -0,0 +1,99 @@
+= ADR-XXX Add Table Description in View DSL
+
+== Context
+
+Add table description to View DSL.
+
+== Decision
+
+=== View DSL
+
+* *Name*: _view#Identifier_, name of the representation.
+* *Label Expression*: _AQL expression_, default name for an instance of this representation.
+* *Icon URLs Expression*: _AQL expression_, icons for this representation.
+* *Description Expression*: _AQL expression_, default description for an instance of this representation.
+* *Precondition Expression*: _AQL expression_, predicate to create representation.
+* *Domain Type*: _view#DomainType_, type of element on which a representation can be created.
+* *Is Stripe Row Semantic Elements Expression*: _AQL expression_, enable or disable background color fading on lines.
+* *Column Descriptions*: _containment multivalued_
+** *Column Header Expression*: _AQL expression_, label for the column header.
+** *Column Semantic Elements Expression*: _AQL expression_, element to display in the form of column.
+** *Column Label Expression*: _AQL expression_, label for the column.
+** *Column Icon URLs Expression*: _AQL expression_, icons for the column.
+** *Column Initial Width Expression*: _AQL expression_, initial width to apply on the column.
+** *Column Can Resize Expression*: _AQL expression_, enable or disable the resize for the column.
+** *Column Filter Widget Expression*: _AQL expression_, widget type to apply as filter on the column.
```

Comment:
I'm very curious what the result of this expression should be because I don't think a simple string can be enough to define how to perform filtering. We can have for example range based filtering with a minimum and a maximum.

Is it enough for the scope of the feature we need? Wouldn't that require a dedicated ADR? (it may very well be enough but I'd like you to be sure)